### PR TITLE
Add task and show task list

### PR DESCRIPTION
**Pull request by ledinhhuan**
Store task and show tasks list.

### CODE REVIEW

**Comment by ledinhhuan** [comment]
ready

File: config/setting.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'number' => 0,
+];
```
Comment (tranducquoc) [inline config/setting.php]: end file

**Comment by ledinhhuan** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**